src/components/QuickRunParametersModal.vue

```diff
@@ -25,7 +26,7 @@
 
     <template #actions>
       <slot name="actions">
-        <p-button type="submit" primary :form="formId">
+        <p-button type="submit" :disabled="loading" primary :form="formId">
```

Comment:
```suggestion
        <p-button type="submit" :loading primary :form="formId">
```
any reason not to use the loading state here? that should also disable the button